src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -495,7 +495,7 @@ impl HummockVersion {
                             .clone_from(&group_construct.table_ids);
                         self.levels.insert(*compaction_group_id, new_levels);
                         let member_table_ids = if group_construct.version
-                            >= CompatibilityVersion::NoMemberTableIds as _
+                            >= CompatibilityVersion::NoMemberTableIds as i32
```

Comment:
avoid unrelated change

Reply:
It seems we do need this as long as we use `jiff` in `src/common/src/types/datetime.rs`. Not sure why though ... maybe a rustc bug.

Btw it can be `group_construct.version() >= CompatibilityVersion::NoMemberTableIds` rather than `if group_construct.version >= CompatibilityVersion::NoMemberTableIds as i32`

Reply:
It doesn't seem to work because the type of CompatibilityVersion::NoMemberTableIds is not i32

Reply:
`group_construct.version()` returns `CompatibilityVersion`, while `group_construct.version` is `i32`